### ISSUE TITLE
TRI CORS/404 error

### ISSUE THREAD

**Issue by jywarren** [publiclab/leaflet-environmental-layers#514]
I'm now seeing this on https://publiclab.org/unearthing-pvd:

```
Access to XMLHttpRequest at 'https://iaspub.epa.gov/enviro/efservice/tri_facility/pref_latitude/BEGINNING/41/PREF_LONGITUDE/BEGINNING/71/rows/0:300/JSON?_=1615315565558' from origin 'https://publiclab.org' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

I believe that's actually a 404 error not CORS: the URL directly gives: `The requested URL /enviro/efservice/tri_facility/pref_latitude/BEGINNING/41/PREF_LONGITUDE/BEGINNING/71/rows/0:300/JSON was not found.`

https://iaspub.epa.gov/enviro/efservice/tri_facility/pref_latitude/BEGINNING/41/PREF_LONGITUDE/BEGINNING/71/rows/0:300/JSON?_=1615315565558

**Comment by jywarren** [comment]
I guess we can assume this EPA TRI API is just offline now. There must be a new or better way to access it. Related:

#46 #393 #43

**Comment by jywarren** [comment]
Merge into https://github.com/publiclab/leaflet-environmental-layers/issues/393
